.github/workflows/dapr-test.yml

```diff
@@ -507,6 +507,7 @@ jobs:
           make setup-test-env-redis
           make setup-test-env-kafka
           make setup-test-env-zipkin
+          make setup-test-env-temporal
```

Comment:
Just FYI this seems to take about 3-4 mins for me on AKS. Although out of scope for this PR, it would be nice if we could find ways to either reduce the number of pods this deploys (can we do it without Cassandra for example?) and/or maybe deploy all 3rd party stuff (Redis, Kafka, MongoDB) in parallel.

Reply:
Unfortunately, this is with only required dependencies: https://artifacthub.io/packages/helm/wener/temporal